README.md

```diff
@@ -77,6 +77,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
+| [LoanDolphin](https://loandolphin.com.au/) | Home loan marketplace. | `Lisboa` `Remote` |
```

Comment:
Please switch order with Feedzai.  
Companies must be sorted alphabetically.

Thanks!

Reply:
Missed having PRs reviewed by you buddy. :)

Reply:
@marmelo done 👍